index.mdx

```diff
@@ -1,7 +1,7 @@
-# Welcome to iClasser Documentation
-
-Welcome to the documentation for iClasser.
-
-## Getting Started
-
-- [Introduction](/docs/getting-started)
+# Welcome to iClasser Documentation
+
+Welcome to the documentation for iClasser.
+
+## Getting Started
+
+- [Introduction](/docs/getting-started)

```

Comment:
Thank you for raising PR @Khalidnoori568 
Lets make links like this for each section we built.
Currently there is one `getting-started.mdx` file, lets have multiple files for each one, and will have sub sections.

Reply:
For example we can create `types-of-accounts.mdx` => for Types of accounts section like this:
\- \[Types of accounts\]\(\/docs\/types-of-accounts\)